core/arch/arm/plat-imx/crypto_conf.mk

```diff
@@ -46,8 +47,14 @@ $(call force, CFG_NXP_CAAM_RUNTIME_JR, y)
 #
 # Definition of all HW accelerations for all i.MX
 #
+ifeq ($(CFG_CORE_SE05X), y)
+$(call force, CFG_NXP_CAAM_RNG_DRV, n)
+CFG_CRYPTO_DRIVER_DEBUG = 0
+else
 $(call force, CFG_NXP_CAAM_RNG_DRV, y)
-$(call force, CFG_WITH_SOFTWARE_PRNG,n)
+endif
+
+CFG_WITH_SOFTWARE_PRNG = n
```

Comment:
ok

Reply:
@ldts , 
can you try not add any SE050 specific flag in this file.
if you need to disable CAAM RNG driver because you would like (i guess) SE050 to handle, what about setting the CFG_NXP_CAAM_RNG_DRV as an explicit default value (set to yes) and then keep the  $(call force, CFG_WITH_SOFTWARE_PRNG,n) if the CFG_NXP_CAAM_RNG_DRV  is set to yes.

that way if in your build you add CFG_CORE_SE05X you can also pass CFG_NXP_CAAM_RNG_DRV  to n

Reply:
@sdininno yes, that is cleaner. will do that.